airflow/serialization/serialized_objects.py

```diff
@@ -1068,6 +1068,7 @@ def detect_task_dependencies(task: Operator) -> list[DagDependency]:
                 DagDependency(
                     source=task.dag_id,
                     target=getattr(task, "trigger_dag_id"),
+                    label=task.task_display_name or task.task_id,
```

Comment:
I think `task_display_name` already handles the `or task_id` logic so this is not needed (the `or` never fires)

```suggestion
                    label=task.task_display_name,
```

Reply:
got it. will change them back. Thanks!